test/spec/modules/pubmaticBidAdapter_spec.js

```diff
@@ -124,6 +126,59 @@ describe('PubMatic adapter', function () {
       }
     ];
 
+    nativeBidRequests = [{
+      code: '/19968336/prebid_native_example_1',
+      sizes: [
+        [300, 250]
+      ],
+      mediaTypes: {
+        native: {
+          title: {
+            required: true,
+            length: 50
+          },
+          image: {
+            required: true,
+            sizes: [300, 250]
+          }
+        }
+      },
+      nativeParams: {
+        title: { required: true }
+      },
+      bidder: 'pubmatic',
+      params: {
+        publisherId: '5670',
+        adSlot: 'div-1',
+      }
+    }];
+
+    nativeBidRequestsWithoutAsset = [{
+      code: '/19968336/prebid_native_example_1',
+      sizes: [
+        [300, 250]
+      ],
+      mediaTypes: {
+        native: {
+          title: {
```

Comment:
we are supporting more assets than handled in case here

Reply:
Yes 
I've written test cases for what prebid support now 
and rest will be given back from server but will not be sent to dfp as there are no mapping keys in prebid